### ISSUE TITLE
use localtunnel (or something like it) to expose localhost-based URLs to the internet

### ISSUE THREAD

**Issue by csells** [encounterplus/encounterplus#180]
client code: https://github.com/localtunnel/localtunnel
server code: https://github.com/localtunnel/server

**Comment by csells** [comment]
Here's another potential service for this purpose: https://beyondco.de/docs/expose/introduction

**Comment by jurex** [comment]
Good idea, but all of these services are for destkop. I'm trying to find a native mobile alternative. Thanks!
